src/runtimes/go/builder.js

```diff
@@ -0,0 +1,33 @@
+const { basename } = require('path')
+
+const { lstat } = require('../../utils/fs')
+const { runCommand } = require('../../utils/shell')
+
+const build = async ({ destPath, mainFile, srcDir }) => {
+  const functionName = basename(srcDir)
+
+  try {
+    await runCommand('go', ['build', '-o', destPath, '-ldflags', '-s -w'], {
+      cwd: srcDir,
+      env: {
+        GOOS: 'linux',
+      },
+    })
```

Comment:
consider setting `CGO_ENABLED=0` so that people get a failure when trying to include native libs which likely won't work well on lambda.

giving an option to enable if they are sure might be nice.
or if you allow people to pass arbitrary env vars to this command, consider setting `CGO_ENABLED=0` if there is no `CGO_ENABLED` given

Reply:
I'd like to keep this initial PR simple and introduce any configuration properties in follow up PRs. Perhaps we can hardcode `CGO_ENABLED=0` for now, and then study the best way to make this configurable. Does that sound reasonable?

Reply:
Done in https://github.com/netlify/zip-it-and-ship-it/pull/576/commits/23afdea7c21455bfb7397d275b799ce95762d97e.